### PR TITLE
Bug 1795713: Update filter help message

### PR DESCRIPTION
**Pull request by zherman0**
Update filter help message to indicate what should be entered into field.
Screenshot_2020-01-29 Search · OKD
Screenshot_2020-01-29 Search · OKD(1)

### CODE REVIEW

**Comment by zherman0** [comment]
@benjaminapetersen - Does this work for you?
@bmignano @spadgett - any better examples to use for name instead of (Example: resource-123)

**Comment by bmignano** [comment]
LGTM, thanks Zac!

File: frontend/public/components/search-filter-dropdown.tsx [hunk 1]
```diff
@@ -52,7 +52,9 @@ export const SearchFilterDropdown: React.SFC<SearchFilterDropdownProps> = (props
         <TextInput
           onChange={setInputValue}
           placeholder={
-            selected === searchFilterValues.Label ? 'Filter by label...' : 'Filter by name...'
+            selected === searchFilterValues.Label
+              ? 'Filter by label (Example: app=frontend)'
+              : 'Filter by name (Example: resource-123)'
```
Comment (spadgett) [inline frontend/public/components/search-filter-dropdown.tsx]:
```suggestion
              : 'Filter by name (Example: my-resource)'
```

**Comment by benjaminapetersen** [comment]
I would tend to agree, `app=frontend` or `my-resource` is plenty.

**Comment by zherman0** [comment]
This overlaps the work being done in PR4140 so I am closing this.

**Pull request closed without merging**